include/elf/arc-reloc.def

```diff
@@ -497,3 +497,24 @@ ARC_RELOC_HOWTO(ARC_NPS_CMEM16, 78, \
                 replace_bits16, \
                 dont, \
                 ( ME ( S + A )))
+
+ARC_RELOC_HOWTO(ARC_JLI_SECTOFF, 63, \
+                2, \
+                10, \
+                replace_jli, \
+                bitfield, \
+                ( ( S - JLI ) ))
+
+ARC_RELOC_HOWTO(ARC_JLI_SECTOFF_SIMM12, 79, \
+                2, \
+                12, \
+                replace_add2_jli, \
+                bitfield, \
+                ( ( S - JLI ) ))
+
+ARC_RELOC_HOWTO(ARC_JLI_32, 80, \
+                2, \
+                32, \
+                replace_word32_jli, \
+                bitfield, \
+                ( ( S - JLI ) ))
```

Comment:
As far as I can see this formula combined with the op in replace_word32_jli says that ARC_JLI_32 is actually (S - JLI) + JLI = S. Is it correct?